### PR TITLE
Book room modal integration

### PR DESCRIPTION
**Pull request by alfredbrannare**
Added my eventListener inside the forEach loop inside load-challenges.js

### CODE REVIEW

**Review by itsJoel59 — APPROVED** [review]
Looks good ok to merge

**Review by Paajt — APPROVED** [review]
I agree with Joel, looks good! Ok to merge!

File: load-challenges.js [hunk 1]
```diff
@@ -20,6 +20,17 @@ async function cardAPI() {
             clone.querySelector(".onsite-online").textContent = challenge.type;
             clone.querySelector(".ppl").textContent = `${challenge.minParticipants}-${challenge.maxParticipants} participants`;
             clone.querySelector(".describe").textContent = challenge.description;
+            const cardButton = clone.querySelector("button");
+
+            cardButton.addEventListener("click", () => {
+                challengePeople = [];
+                
+                for (let i = challenge.minParticipants; i <= challenge.maxParticipants; i++) {
+                    challengePeople.push(i);
```
Comment (Emilyempa) [inline load-challenges.js:29]: Nice job to add this in the function.

Comment (alfredbrannare) [reply load-challenges.js:29]: Thanks!

**Review by Emilyempa — APPROVED** [review]
Nice work with getting API with avalable dates, times and participants

**Pull request merged**